client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -29,15 +30,31 @@ object FlightTableComponents {
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  def pcpTimeRange(fws: ApiFlightWithSplits): TagOf[Div] =
+  def actualMinutesToChox(arrival: Arrival): Option[MillisSinceEpoch] =
+    arrival.ActualChox.flatMap(c => arrival.Actual.map(a => (c - a) / oneMinuteMillis))
+
+  def estMinutesToChox(arrival: Arrival): Option[MillisSinceEpoch] =
+    arrival.EstimatedChox.flatMap(c => arrival.Actual.map(a => (c - a) / oneMinuteMillis))
+
+  def pcpTimeRange(fws: ApiFlightWithSplits, firstPaxOffMillis: MillisSinceEpoch): VdomElement =
     fws.apiFlight.PcpTime.map { pcpTime: MillisSinceEpoch =>
       val sdateFrom = SDate(MilliDate(pcpTime))
       val sdateTo = SDate(MilliDate(pcpTime + millisToDisembark(fws.pcpPaxEstimate.pax.getOrElse(0))))
-      <.div(^.display := "flex", ^.flexWrap := "nowrap",
-        sdateLocalTimePopup(sdateFrom),
+      val postTouchdownTimes = <.span(
+        <.h3("Minutes to chox from touchdown"),
```

Comment:
We are listing down more than chox from touchdown so may be h3 should be Arrival hall from touchdown ?

Reply:
There's another h3 with 'Other times' below this one.